packages/@webex/plugin-cc/src/WebSocket/WebSocket.ts

```diff
@@ -0,0 +1,151 @@
+import Mercury from '@webex/internal-plugin-mercury';
+import {HTTP_METHODS, WebSocketEvent} from '../types';
+import webSocketConfig from './config';
+import IWebSocket from './IWebSocket';
+
+class WebSocket extends (Mercury as any) implements IWebSocket {
+  /**
+   * @instance
+   * @memberof WebSocket
+   * @private
+   * @type {string}
+   */
+  private webSocketUrl: string;
+
+  /**
+   * @instance
+   * @memberof WebSocket
+   * @private
+   * @type {string}
+   */
+  private subscriptionId: string;
+
+  /**
+   * @instance
+   * @memberof WebSocket
+   * @private
+   * @type {string}
+   */
+  private datachannelUrl: string;
+
+  constructor(options: object = {}) {
```

Comment:
As mentioned earlier, please avoid using any or object as types. Either add proper types or add TODO comment to add a type in future stories if it's out of this PR's scope

Reply:
addressed